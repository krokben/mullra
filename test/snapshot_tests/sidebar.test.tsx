import * as React from "react";
import { create } from "react-test-renderer";

import Sidebar from "../../src/components/sidebar";

describe("Sidebar", () => {
  test("snapshot renders", () => {
    const root: any = create(<Sidebar />);
    expect(root.toJSON()).toMatchSnapshot();
  });
});
