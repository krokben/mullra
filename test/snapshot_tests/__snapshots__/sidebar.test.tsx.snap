// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar snapshot renders 1`] = `
<ul
  className="sc-bdVaJa cPkZj"
>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
  <li>
    Component
  </li>
</ul>
`;
